pipeline/README.md

```diff
@@ -3,8 +3,11 @@
 If you want to run Bats tests locally (instead of in the CI pipeline), execute the following from the project root directory:
 
 ```bash
+git submodule update --recursive --init # init and update bats helpers
+
 docker build -t ck8s-apps ./pipeline
-docker run --rm -it -v "$(pwd):/ck8s-apps:ro" -w /ck8s-apps ck8s-apps \
+docker run --rm -it -e CK8S_PGP_FP=529D964DE0BBD900C4A395DA09986C297F8B7757 \
```

Comment:
Who's fingerprint is this?

Reply:
It is the same as in the workflow file [here](https://github.com/elastisys/compliantkubernetes-apps/blob/main/.github/workflows/pipeline.yml#L73), it was previously found [here](https://github.com/elastisys/compliantkubernetes-apps/commit/7935222afbb6695a4a5406bd15d1e3e75431a725#diff-ac4a92945bb0aa4c42cfd49d29a88c433c8a52b85bded34032cc66184271ea2dL16)

Reply:
Ah. Do you know why it needs to be set there when it's being set in the `init.bats` file as well?

Reply:
Sorry if I wasn't clear, it is no longer set in `init.bats`, it got moved  to the workflow file, so when you try to run it locally one of the tests fails if you do not provide it as an environment variable.